front/app/modules/commercial/segment/index.tsx

```diff
@@ -94,7 +94,7 @@ const configuration: ModuleConfiguration = {
                 locale: user.data.attributes.locale,
                 isSuperAdmin: isSuperAdmin(user),
                 isAdmin: isAdmin(user),
-                isProjectModerator: isProjectModerator(user),
+                isProjectModerator: isModerator(user),
```

Comment:
When no projectId was defined as is here the case, `isProjectModerator` just checked `isModerator`. Not ideal, but not going to change it now.